proto/api/v1/auth.proto

```
@@ -0,0 +1,178 @@
+syntax = "proto3";
+
+package api.v1;
+
+import "buf/validate/validate.proto";
```

Buf (1.54.0)

import "buf/validate/validate.proto": file does not exist

(COMPILE)